SzczepionkaAPI/src/main/java/com/szczepionka/util/PatientLinkGenerator.java

```diff
@@ -0,0 +1,32 @@
+package com.szczepionka.util;
+
+import org.springframework.stereotype.Component;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.UUID;
+
+@Component
```

Comment:
Why does util class is marked as a component?

Reply:
changed.